core/spec.md

```diff
@@ -3692,8 +3688,8 @@ Where:
   of the 3 `RESOURCE` attributes MUST follow these rules:
```

Comment:
The dot at the end of the previous line needs to be removed.

Reply:
done